Index: p0f/config.h
===================================================================
--- p0f.orig/config.h	2014-01-14 17:37:14.381363938 +0100
+++ p0f/config.h	2014-01-17 09:34:19.391081189 +0100
@@ -20,7 +20,7 @@
 /* Default location of p0f.fp: */
 
 #ifndef FP_FILE
-#  define FP_FILE           "p0f.fp"
+#  define FP_FILE           "/etc/p0f/p0f.fp"
 #endif /* !FP_FILE */
 
 /* Initial permissions on log files: */
